tests/models/depth_anything/test_modeling_depth_anything.py

```diff
@@ -236,26 +236,24 @@ def _validate_backbone_init():
 
 
 # We will verify our results on an image of cute cats
-def prepare_img():
-    image = Image.open("./tests/fixtures/tests_samples/COCO/000000039769.png")
-    return image
+def predict_depth(image_processor, model):
+    image = image = Image.open("./tests/fixtures/tests_samples/COCO/000000039769.png")
+    inputs = image_processor(images=image, return_tensors="pt").to(torch_device)
+    with torch.no_grad():
+        outputs = model(**inputs)
+    return outputs.predicted_depth
 
 
 @require_torch
 @require_vision
 @slow
 class DepthAnythingModelIntegrationTest(unittest.TestCase):
     def test_inference(self):
+        # -- `relative` depth model
         image_processor = DPTImageProcessor.from_pretrained("LiheYoung/depth-anything-small-hf")
         model = DepthAnythingForDepthEstimation.from_pretrained("LiheYoung/depth-anything-small-hf").to(torch_device)
 
-        image = prepare_img()
-        inputs = image_processor(images=image, return_tensors="pt").to(torch_device)
-
-        # forward pass
-        with torch.no_grad():
-            outputs = model(**inputs)
-            predicted_depth = outputs.predicted_depth
+        predicted_depth = predict_depth(model, image_processor)
```

Comment:
Hey @bt2513, first time contributor here. I was running your code locally and found that the order of args is not correct here.

Reply:
Thanks for catching this! I made some last min cleanup before pushing and missed it 😅